app/sdil/models/Pages.scala

```diff
@@ -240,19 +243,27 @@ case object StartDatePage extends MidJourneyPage {
 }
 
 case object ProductionSitesPage extends MidJourneyPage {
-  override def nextPage(formData: RegistrationFormData): Page = WarehouseSitesPage
-
-  override def previousPage(formData: RegistrationFormData): Page = StartDatePage
+  override def nextPage(formData: RegistrationFormData)(implicit config: AppConfig): Page = WarehouseSitesPage
+
+  override def previousPage(formData: RegistrationFormData)(implicit config: AppConfig): Page = {
+    val showStartDate = LocalDate.now isAfter config.taxStartDate
```

Comment:
if this logic has moved here, it should be removed from the ProductionSiteController (and the same for the other controllers that potentially interact with the start date page)